lib/ansible/modules/network/fortimanager/fmgr_provisioning.py

```diff
@@ -0,0 +1,328 @@
+            'os_ver': '5.0',
+            'os_type': 'fos',
+            # 'patch': '0',
+            'desc': 'Provisioned by Ansible',
+        }
+'''
+
+
+    )
+
+    module = AnsibleModule(argument_spec, supports_check_mode=True, )
+
+    # check if params are set
+    if module.params["platform"] is None or module.params["os_ver"] is None or module.params["os_type"] is None:
+        module.fail_json(msg="Additional parameters are required for connection")
+
+    # check if params are set
+    if module.params["host"] is None or module.params["username"] is None:
+        module.fail_json(msg="Host and username are required for connection")
+
+    if module.params["policy_package"] is None:
+        module.params["policy_package"] = 'default'
+    if module.params["adom"] is None:
+        module.params["adom"] = 'root'
+    if module.params["vdom"] is None:
+        module.params["vdom"] = 'root'
+
+    # check if login failed
+    fmg = AnsibleFortiManager(module, module.params["host"], module.params["username"], module.params["password"])
+    response = fmg.login()
+
+    #print(response)
+    #sys.exit()
+
+    if "FortiManager instance connnected" not in str(response):
+        module.fail_json(msg="Connection to FortiManager Failed")
+    else:
+
+        results = create_model_device(fmg,
+                                      module.params["name"],
+                                      module.params["serial"],
+                                      module.params["group"],
+                                      module.params["platform"],
+                                      module.params["os_ver"],
+                                      module.params["os_type"],
+                                      module.params["minor_release"],
+                                      module.params["patch_release"],
+                                      module.params["adom"])
+        if not results[0] == 0:
+            module.fail_json(msg="Create model failed", **results)
+
+        results = update_flags(fmg, module.params["name"])
+        if not results[0] == 0:
+            module.fail_json(msg="Update device flags failed", **results)
+
+        # results = assign_dev_grp(fmg, 'Ansible', 'FGVM000000117992', 'root', 'root')
+        # if not results[0] == 0:
+        #     module.fail_json(msg="Setting device group failed", **results)
+
+        results = update_install_target(fmg, module.params["name"], module.params["policy_package"])
+        if not results[0] == 0:
+            module.fail_json(msg="Adding device target to package failed", **results)
+
+        results = install_pp(fmg, module.params["name"],module.params["policy_package"])
+        if not results[0] == 0:
+            module.fail_json(msg="Installing policy package failed", **results)
+
+        fmg.logout()
+
+        # results is returned as a tuple
+        return module.exit_json(**results[1])
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Missing new line

Reply:
Hi @gundalow - I've fixed this and all the other syntax errors.  Thanks